docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -217,3 +223,50 @@ attachment:
       template_upload
   code: |
     pdf_fields
+---
+attachment:
+  variable name: test_assemble_pdf
+  pdf template file:
+    code: |
+      template_upload
+  fields:
+    - "Field1": "Value1"
+---
+code: |
+  from pdfminer.pdfparser import PDFSyntaxError
+  from PyPDF2.utils import PdfReadError
+
+  try:
+    pdf_concatenate(test_assemble_pdf)
+  except PDFSyntaxError:
+    # Handle PDF read error
+    exit_invalid_file
+  except PdfReadError:
+    exit_xref_table
+  except:
+    pass # Continue on to safer checks with get_pdf_fields
+  validate_pdf = True
+---
+event: exit_xref_table
+question: |
+  This PDF is not valid
+subquestion: |
+  There was a problem reading this PDF.
+  
+  Sometimes you can fix this problem by using 
+  [documate.org/pdf](https://www.documate.org/pdf) or by using 
+  [qpdf](http://qpdf.sourceforge.net/).
+  
+  [Read 
+  more](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/pdfs#corrupted-or-locked-pdfs).
+buttons:
+  - Restart: restart
+---
+event: exit_something_went_wrong
+question: |
+  Something went wrong reading your template
+subquestion: |
+  Are you sure this is a valid document?
+buttons:
+  - Restart: restart
```

Comment:
When does this happen? Can't see it referenced anywhere else.

Reply:
Good catch--I had a fallback catchall condition, but that blocks NameError so it's not really usable and I removed the trigger for it. I'll take this out too.